admin/openwisp-monitoring/Makefile

```diff
@@ -0,0 +1,115 @@
+# openwisp.org
+#
+# This is free software, licensed under the GNU General Public License v3.
+# See /LICENSE for more information.
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openwisp-monitoring
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Federico Capoano <support@openwisp.io>
+PKG_LICENSE:=GPL3.0-or-later
+
```

Comment:
PKG_LICENSE_FILES:=LICENSE

Reply:
Shouldn't we maintain consistency with the already existing openwisp-config package?
https://github.com/openwrt/packages/blob/master/admin/openwisp-config/Makefile#L11

Reply:
He was trying to tell you if there is a licence file in your repository.
https://github.com/openwisp/openwrt-openwisp-monitoring/blob/master/LICENSE